src/router/index.js

```diff
@@ -1,20 +1,25 @@
 import React from 'react'
-import { StaticRouter } from 'react-router'
+import { StaticRouter, Switch, Route } from 'react-router'
 import { ConnectedRouter } from 'connected-react-router'
 import { createBrowserHistory, createMemoryHistory } from 'history'
+import HyloAppRouter from 'routes/HyloAppRouter'
 import RootRouter from 'routes/RootRouter'
 import '../css/global/index.scss'
 
 export const history = typeof window !== 'undefined'
   ? createBrowserHistory()
   : createMemoryHistory()
 
+// Note: `/hyloApp/*` routes will not invoke auth as auth cookie is already passed from webview
 export function clientRouter () {
   require('client/rollbar') // set up handling of uncaught errors
 
   return (
     <ConnectedRouter history={history}>
-      <RootRouter />
+      <Switch>
+        <Route path='/hyloApp' component={HyloAppRouter} />
```

Comment:
Adding top-level pre-auth check route for Editor and potentially other future WebView'd items like it. This makes the Editor load notably quicker, good enough, but still not as quickly as I'd like to see.  * This ideally would be server-side rendered (SSR) but our current (pretty sure non-functional) SSR is in the way of doing that simply for this component.